tests/doctests/wfs_MapServerWFSCapabilities.txt

```diff
@@ -101,6 +101,6 @@ Test exceptions
 Lastly, test the getcapabilities method
 
     >>> wfs = WebFeatureService('http://nsidc.org/cgi-bin/atlas_south?', version='1.0')
-    >>> xml = wfs.getcapabilities().read().decode('utf-8')
+    >>> xml = wfs.getcapabilities().read().decode('ISO-8859-1')
     >>> xml.find('<WFS_Capabilities') > 0
```

Comment:
In this case:

```
xml = wfs.getcapabilities().read()
xml.find('<WFS_Capabilities') > 0
```

you will get an error `TypeError("Type str doesn't support the buffer API",)` for Python3.